File: vseros/2016-17/okr/4.py
n = int(input())
p = int(input())
k = int(input()) % n

a = (p - 2 + 2 * k * (p % 2 * 2 - 1)) % n + 1
b = (p     + 2 * k * (p % 2 * 2 - 1)) % n + 1 
print(min(a, b), max(a, b))

File: vseros/2016-17/okr/4.cpp
#include <iostream>

int main()
{
	int n, p, k;
	cin >> n >> p >> k;

	int pos1 = (p + 2 * k * (p % 2 * 2 - 1)) % n + 1;
	int pos2 = (p - 2 + 2 * k * (p % 2 * 2 - 1)) % n + 1;
	cout << min(pos1, pos2) << ' ' << max(pos1, pos2) << endl;
}
